pkg/crypto/certloader/fswatcher/fswatcher.go

```diff
@@ -343,11 +343,13 @@ func (w *Watcher) loop() {
 				}
 			}
 		case err := <-w.watcher.Errors:
-			log.WithError(err).Debug("Received fsnotify error")
+			log.WithError(err).Warn("Received fsnotify error while watching")
```

Comment:
watching what? is my immediate question. Does the error message contain all relevant info?

Reply:
I would _guess_ the error message has enough info, but I've added "while watching" just to clarify that it is not an initialization nor a cleanup error.